microsoft-edge/dualengine/concepts/adapter-dll.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Understanding the DualEngine Adapter DLL
+description: Understanding the requirements and uses of DualEngine Adapter DLL.
+author: MSEdgeTeam
+ms.author: edgededev
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.localizationpriority: high
+ms.technology: windows-integration
+ms.date: 11/06/2023
+---
+
+# Understanding the DualEngine Adapter DLL
+The DualEngine API is an API that can only be used from inside the Internet Explorer process.
+As such in order to make use of the API it is necessary to create a plugin DLL that Internet Explorer will load.
+This DLL will host your code that uses the DualEngine API as well as presumably any code you need to communicate
+with your main application, hence why it is referred to as the adapter.
+
+There are a number of requirements to fulfill to have Internet Explorer successfully load your DLL.
+
+## Limited Access Feature Requirements
+The DualEngine API is what is known as a Limited Access Feature (LAF) which are features that need to be unlocked before
+they can be used, see [LimitedAccessFeatures Class](/uwp/api/windows.applicationmodel.limitedaccessfeatures) for

```

Comment:
```suggestion
it can be used.  See [LimitedAccessFeatures Class](/uwp/api/windows.applicationmodel.limitedaccessfeatures) for
```

Reply:
The DualEngine API is a Limited Access Feature (LAF); that is, a feature that needs to be unlocked before it can be used.  For more information, see [LimitedAccessFeatures Class](/uwp/api/windows.applicationmodel.limitedaccessfeatures).